src/Avalonia.Controls/Primitives/SelectingItemsControl.cs

```diff
@@ -139,6 +139,17 @@ static SelectingItemsControl()
             IsSelectedChangedEvent.AddClassHandler<SelectingItemsControl>((x, e) => x.ContainerSelectionChanged(e));
         }
 
+        /// <summary>
+        /// Event that should be raised by items that implement <see cref="ISelectable"/> to
+        /// notify the parent <see cref="SelectingItemsControl"/> that their selection state
+        /// has changed.
+        /// </summary>
+        public event EventHandler<RoutedEventArgs>? IsSelectedChanged
```

Comment:
I'm not sure that adding a CLR event handler for this event is helpful - it's an "internal" event used for communication between `SelectingItemsControl` and its items, and shouldn't really be listened for in user code - making it a public CLR event might mean that people start listening for this event instead of `SelectionChanged`.

Reply:
Reverted changes